app/services/provider_interface/application_data_export.rb

```diff
@@ -1,66 +1,74 @@
+          'email' => application.application_form.candidate.email_address,
+          'phone_number' => application.application_form.phone_number,
+          'address_line1' => application.application_form.address_line1,
+          'address_line2' => application.application_form.address_line2,
+          'address_line3' => application.application_form.address_line3,
+          'address_line4' => application.application_form.address_line4,
+          'postcode' => application.application_form.postcode,
+          'country' => application.application_form.country,
+          'recruitment_cycle' => RecruitmentCycle.cycle_name(application.application_form.recruitment_cycle_year),
+          'provider_name' => application.current_provider.name,
+          'provider_code' => application.current_provider.code,
+          'accredited_provider_name' => application.current_accredited_provider&.name,
+          'accredited_provider_code' => application.current_accredited_provider&.code,
+          'course_name' => application.current_course.name,
+          'course_code' => application.current_course.code,
+          'site_name' => application.current_site.name,
+          'site_code' => application.current_site.code,
+          'study_mode' => application.current_course_option.study_mode,
+          'start_date' => application.current_course.start_date,
+          'FIRSTDEG' => application.degrees_completed_flag == 1 ? 'TRUE' : 'FALSE',
+          'qualification_type' => application.first_degree&.qualification_type,
+          'non_uk_qualification_type' => application.first_degree&.non_uk_qualification_type,
+          'subject' => application.first_degree&.subject,
+          'grade' => application.first_degree&.grade,
+          'start_year' => application.first_degree&.start_year,
+          'award_year' => application.first_degree&.award_year,
+          'institution_details' => application.first_degree&.institution_name,
+          'equivalency_details' => replace_smart_quotes(application.first_degree&.composite_equivalency_details),
+          'awarding_body' => nil, # included for backwards compatibility. This column is always blank
+          'gcse_qualifications_summary' => replace_smart_quotes(application.gcse_qualifications_summary),
+          'missing_gcses_explanation' => replace_smart_quotes(application.missing_gcses_explanation),
+          'disability_disclosure' => application.application_form.disability_disclosure,
+        }
+      end
+
+      def replace_smart_quotes(text)
+        text&.gsub(/(“|”)/, '"')&.gsub(/(‘|’)/, "'")
+      end
 
-    def self.replace_smart_quotes(text)
-      text&.gsub(/(“|”)/, '"')&.gsub(/(‘|’)/, "'")
+      def rejection_reasons(application_choice)
+        reasons = RejectedApplicationChoicePresenter.new(application_choice).rejection_reasons
+        reasons = reasons.transform_values & :compact
```

Comment:
Rubocop gave me this syntax, I started out with `reasons.transform_values { |arr| arr.reject { |ele| ele.nil? } }`!